security/src/main/java/nl/nn/adapterframework/lifecycle/servlets/SecuritySettings.java

```diff
@@ -0,0 +1,58 @@
+/*
+   Copyright 2023 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.lifecycle.servlets;
+
+import javax.servlet.annotation.ServletSecurity.TransportGuarantee;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.core.env.PropertyResolver;
+
+import lombok.extern.log4j.Log4j2;
+import nl.nn.adapterframework.util.EnumUtils;
+
+@Log4j2
+public class SecuritySettings {
```

Comment:
Does this class have to be static? Or can this be Spring auto-wired?

Reply:
Hij zou static kunnen, ik heb er overnagedacht maar voor nu wat dit even de makkelijkere oplossing (copy-paste naar een andere klasse)